serverless.yml

```diff
@@ -52,6 +52,25 @@ functions:
       NEXT_PUBLIC_API_URL: https://${self:custom.aliases.${self:provider.stage}}/api
       NEXT_PUBLIC_URL: https://${self:custom.aliases.${self:provider.stage}}
 
+  hn-shared-plan-reminders:
+    name: ${self:service}-reminders-${self:provider.stage}
+    handler: build/router/router/cloudwatch/reminders.handler
```

Comment:
This can just be `router/cloudwatch/reminders.handler` now I think?

Reply:
Hm, so I tried that and it didn't seem to work, but I'll try again

Reply:
Yeah this shouldn't reference the build folder - ```router/cloudwatch/reminders.handler``` should be right

Reply:
👌